Source/VMware.vSphereDSC/DSCResources/DRSRule.ps1

```diff
@@ -118,9 +118,10 @@ class DRSRule : BaseDSC {
 
     [void] Set() {
         try {
-            Write-VerboseLog -Message $this.SetMethodStartMessage -Arguments @($this.DscResourceName)
             $this.ConnectVIServer()
 
+            $this.WriteLogUtil('Verbose', $this.SetMethodStartMessage, @($this.DscResourceName))
```

Comment:
Optional suggestion: As the connection is no longer required for the logs, revert the messages to occur before establishing the connection and after the connection was closed.